dev-python/sortedcontainers/sortedcontainers-2.1.0.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 python3_{5,6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python library to sort collections and containers"
+HOMEPAGE="http://www.grantjenks.com/docs/sortedcontainers/"
+SRC_URI="https://github.com/grantjenks/python-${PN}/archive/v${PV}.tar.gz"
```

Comment:
(Additional ` -> ${P}.tar.gz` would have been needed here or the filename would be `v2.1.0.tar.gz` which is not ideal in a single folder on a mirror.)

Reply:
Aah yes, I see, that would indeed have been better. I will take that into account in future ebuilds.